packages/shared/src/hooks/feed/useFeedOnPostClick.ts

```diff
@@ -20,19 +24,26 @@ export default function useFeedOnPostClick(
   columns: number,
   feedName: string,
   ranking?: string,
+  event = 'click',
 ): FeedPostClick {
+  const { postModalByDefault } = useContext(FeaturesContext);
   const { incrementReadingRank } = useIncrementReadingRank();
   const { trackEvent } = useContext(AnalyticsContext);
 
-  return async (post, index, row, column): Promise<void> => {
+  return async (post, index, row, column, optional): Promise<void> => {
     trackEvent(
-      postAnalyticsEvent('click', post, {
+      postAnalyticsEvent(event, post, {
         columns,
         column,
         row,
-        ...feedAnalyticsExtra(feedName, ranking),
+        ...feedAnalyticsExtra(feedName, ranking, { modal: postModalByDefault }),
```

Comment:
This additional param isn't needed at the moment. But thanks for thinking about it :)